tests/cube_fit_test.py

```diff
@@ -0,0 +1,78 @@
+from cube_fit.cube_fit import *
+import numpy as np
+
+
+def six_point_cube_transform_test(face_points_3: np.ndarray, face_points_2: np.ndarray, face_points_1: np.ndarray):
+    '''
+    face_points_3 should contain 3 points in the first plane, 2 for face_points_2 and 1 for face_points_1
+    From face_points_3, we can get the first plane.
+    From face_points_2 and a projection of a point from face_points_2 on the first plane, we can get the second plane
+    From face_points_1, we can get two projections on the previous 2 planes to get the last plane
+    The planes are already othorgonal. Simply arrange them in [x.T,y.T,z.T] to get the frame_transform
+    '''
+    center_3 , normal_3 = plane_of_best_fit(face_points_3)
+    face_points_2 = np.vstack([face_points_2, proj_point_on_plane(face_points_2[1], normal_3, center_3)])
+    center_2, normal_2 = plane_of_best_fit(face_points_2)
+    face_points_1 = np.vstack([face_points_1, proj_point_on_plane(face_points_1[0], normal_3, center_3)])
+    face_points_1 = np.vstack([face_points_1, proj_point_on_plane(face_points_1[0], normal_2, center_2)])
+    distance_to_plane(face_points_1[1], normal_3, center_3)
+    distance_to_plane(face_points_1[2], normal_2, center_2)
+    return find_cube_transform(face_points_3, face_points_2, face_points_1)
```

Comment:
duplicated code aside from a couple of `distance_to_plane` printouts. I would prefer you remove this and just use `six_point_cube_transform` from `cube_fit` directly. Even if it's just a local test, duplicated code is almost always bad for maintainability